app/services/schools/school_metrics_generator_service.rb

```diff
@@ -59,14 +59,11 @@ def suppress_output(&block)
     end
 
     def _suppress_output
-      original_stdout = $stdout.clone
-      original_stderr = $stderr.clone
-      $stderr.reopen File.new('/dev/null', 'w')
-      $stdout.reopen File.new('/dev/null', 'w')
+      original_level = @logger.level
```

Comment:
If we're no longer throwing away $stdout, is there any reason to also reduce the log level from `INFO` to `WARN` in prod?

Reply:
only for less output